.platform.yml

```diff
@@ -1,2 +1,2 @@
 php_settings:
-  version: 7.3
+  version: 8.1
```

Comment:
Actually, this doesn't make any difference because it's a VStack and the parent stack tracks: https://github.com/silverstripe/platform-blank/

Reply:
Hmm. Should we even have a `.platform.yml` file here then?
And what's up with the `infrastructure: ^3` in the platform-blank? Was that legacy? It's not listed in [the docs](https://servicedesk.silverstripe.cloud/support/solutions/articles/75000012884)